docs/iam-structure.md

```diff
@@ -0,0 +1,64 @@
+# IAM Structure
+
+This document aims to outline each of the links in the chain of IAM roles that
+are required for the testing infrastructure. We aim to only use short term
+credentials, and reduce all policies to the minimum required for any given task.
+All of the source code for deploying these roles is in the `infra/` directory.
+
+## Overview
+
+Each of the Prow jobs will first be mounted to a service account. Through
```

Comment:
Couple things...

1) Recommend rewording this first sentence to "Each Prow Job has an associated Kubernetes Service Account (SA) that describes the Kubernetes RBAC context within which the Prow Job's Pod executes." The phrase "first be mounted to a service account" isn't really accurate and the term "mounted" refers to an implementation detail about the service account *token file*.

2) Recommend linking the term "Kubernetes Service Account" to https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/ or https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/